app/controllers/admin/inactive_users_controller.rb

```diff
@@ -31,7 +31,13 @@ def index
     def reactivate
       @user = User.inactive.find(params[:id].to_i)
       pass = Devise.friendly_token(50)
-      @user.update(active: true, expired_at: nil, password: pass, password_confirmation: pass)
+      @user.update(
+        active: true,
+        last_activity_at: Time.now,
```

Comment:
Can we make this Time.current just to be consistent?

Reply:
Done.